web/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+server.port=8081
+
+spring.profiles.active=prod
+
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
Why is it here? And below props also.

Reply:
Maybe it's better to put in the appropriate classes? Active profile in class with @SpringBootApplication annotation, datasource properties in datasource.properties file located in repository resources.

Reply:
don't see 'datasource.properties'